packages/gp2-components/src/molecules/ExpandableText.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Button,
+  chevronCircleDownIcon,
+  colorWithTransparency,
+  lead,
+  pixels,
+} from '@asap-hub/react-components';
+
+import { css } from '@emotion/react';
+import React, { useState } from 'react';
+
+const { rem } = pixels;
+
+const expandableMaxHeight = 120;
+
+const textStyles = css({
+  maxHeight: rem(expandableMaxHeight),
+  overflowY: 'clip',
+  margin: 0,
+});
+
+const expandedTextStyles = css({
+  maxHeight: rem(24 * 250), // aproximation of possible max height (2500 characters) 100 chars per line
```

Comment:
Maybe use the line height const here. Probally in pixels.

Reply:
A simple zoom would mess this kind of count up. Shouldn't we use some line clamp thing or something?